react-native-libraries.json

```diff
@@ -14264,5 +14264,15 @@
      "examples": ["https://github.com/PostHog/posthog-react-native-session-replay/tree/main/example"],
      "ios": true,
      "android": true
-   }
+   },
+  {
+    "githubUrl": "https://github.com/tschoffelen/react-native-check-version",
+    "npmPkg": "https://www.npmjs.com/package/react-native-check-version",
```

Comment:
Hey @pioner92, thanks for adding the new entry to the directory! 👍 

```suggestion
```

The `npmPkg` field expect npm package name, not registry URL, but in case of current entry it can be skipped, since GitHub repository name matches npm package name. The field is mostly dedicated for scoped packages, and correcting this should result in passing CI checks.

Reply:
Thanks @Simek I changed it